.circleci/config.yml

```diff
@@ -72,6 +95,9 @@ workflows:
             - hmpps-approved-premises-api-prod
           requires:
             - request-prod-approval
+      - trigger-ui-tests:
+          requires:
+            - request-prod-approval
```

Comment:
I'm not clear why these tests require prod approval? My understanding was that they run in the dev environment.

Reply:
I was thinking that the api deploys to prod and then the UI tests are run to validate the latest

Reply:
I've changed this to a deploy_dev requirement now. I've tested it by adding the feature branch to the filtered branches list and it triggers the UI builds once the api is deployed to dev